app/assets/sass/patterns/_check-your-answers.scss

```diff
@@ -1,14 +1,75 @@
+
+// please adjust, depending on length of questions and answers and number of sections
+$cya-first-column-width: 30% !default;
+// recommended for single sections: auto
+// recommended for longer questions: 50%
```

Comment:
Suggest adding a comment with the recommendation for shorter questions. That way if a service goes away from the default, our recommendation is still in the comments.

Is it clear what we mean by single sections?